packages/contracts/contracts/Unirep.sol

```diff
@@ -60,6 +60,7 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
     mapping(uint256 => IncrementalTreeData) public globalStateTree;
     // epoch => root => bool
     mapping(uint256 => mapping(uint256 => bool)) public globalStateTreeRoots;
+    mapping(uint256 => uint256) public epochTreeRoots;
```

Comment:
Why do we need this mapping?

Reply:
We have [this function](https://github.com/Unirep/Unirep/blob/main/packages/contracts/contracts/Unirep.sol#L919) that can be used to load epoch tree roots.

Reply:
I think in the future the `epochTree[epoch]` will be destructed?

Reply:
Ah yea, i think it's okay to use the function for now and add this structure once we build that logic